templates/basic_answerfile.txt.j2

```diff
@@ -51,6 +51,9 @@ OVESETUP_DWH_DB/securedHostValidation=none:None
 {% if ovirt_engine_setup_organization is defined %}
 OVESETUP_PKI/organization=str:{{ ovirt_engine_setup_organization }}
 {% endif %}
+{% if ovirt_engine_overwrite_pki_files is defined %}
+QUESTION/1/OVESETUP_PKI_VERIFY_MISSING_CA_PEM=str:{{ ovirt_engine_overwrite_pki_files | ternary('yes','no') }}
```

Comment:
Please use the same answer syntax as other options.
Not sure with location of the answer. In which version was it added?